### PR TITLE
Add LaravelRouteAssertsTrait

### PR DESCRIPTION
**Pull request by jetexe**
| Q             | A
| ------------- | ---
| Bug fix?      |  No
| New feature?  | Yes

## Description

I've add assert that can help to check route list for unexisted classes and methods

Fixes: #23 

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I wrote unit tests for my code
- [x] I have made changes in [CHANGELOG.md](https://github.com/avto-dev/dev-tools/blob/master/CHANGELOG.md) file

### CODE REVIEW

File: src/Tests/PHPUnit/Traits/LaravelRoutesAssertsTrait.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+use Illuminate\Routing\Route;
+use Illuminate\Routing\Router;
+
+trait LaravelRoutesAssertsTrait
+{
+    /**
+     * Assert that all existed routes point to real controllers and actions.
+     */
+    public function assertRoutesActionsExist()
```
Comment (tarampampam) [inline src/Tests/PHPUnit/Traits/LaravelRoutesAssertsTrait.php]: Maybe `assertRoutesActionsExist` -> `assertAllRoutesHasActions`?

File: src/Tests/PHPUnit/Traits/LaravelRoutesAssertsTrait.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    {
+        /** @var Router $router */
+        $router = $this->app->make(Router::class);
+        /** @var Route $route */
+        foreach ($router->getRoutes() as $route) {
+            $controller = $route->getAction()['uses'] ?? null;
+            if (\is_string($controller)) {
+                $class_method = \explode('@', $controller, 2);
+
```
Comment (tarampampam) [inline src/Tests/PHPUnit/Traits/LaravelRoutesAssertsTrait.php]: rm empty line

File: tests/Tests/PHPUnit/Traits/AdditionalAssertionsTraitTest.php [hunk 3]
```diff
@@ -37,7 +37,8 @@ public function testsTraitAsserts()
         $this->makeAssertTest('assertEmptyArray', [[]], ['foo', [1], new \stdClass, [[]]]);
 
         /* @see AdditionalAssertionsTrait::assertIsString */
-        $this->makeAssertTest('assertIsString', ['foo', 'bar'], [null, 1, new class {
+        $this->makeAssertTest('assertIsString', ['foo', 'bar'], [null, 1, new class
```
Comment (tarampampam) [inline tests/Tests/PHPUnit/Traits/AdditionalAssertionsTraitTest.php]: `{` in anonimous classes shold be on same line with class definition (CS rule)

File: tests/Tests/PHPUnit/Traits/AdditionalAssertionsTraitTest.php [hunk 4]
```diff
@@ -133,7 +134,8 @@ public function __toString()
      */
     protected function classUsedTraitFactory()
     {
-        return new class extends \PHPUnit\Framework\TestCase {
+        return new class extends \PHPUnit\Framework\TestCase
```
Comment (tarampampam) [inline tests/Tests/PHPUnit/Traits/AdditionalAssertionsTraitTest.php]: `{` in anonimous classes shold be on same line with class definition (CS rule) [2]

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## v1.8.0
+
+### Added
+
+- Added `LaravelRoutesAssertsTrait` that include `assertRoutesActionsExist` method
```
[the remark this answers is not in the document]
Comment (jetexe) [reply CHANGELOG.md]: Done

File: src/Tests/PHPUnit/Traits/LaravelRoutesAssertsTrait.php (hunk 1 above)
Comment (jetexe) [reply src/Tests/PHPUnit/Traits/LaravelRoutesAssertsTrait.php]: Done

File: src/Tests/PHPUnit/Traits/LaravelRoutesAssertsTrait.php (hunk 2 above)
Comment (jetexe) [reply src/Tests/PHPUnit/Traits/LaravelRoutesAssertsTrait.php]: Done

File: tests/Tests/PHPUnit/Traits/AdditionalAssertionsTraitTest.php (hunk 3 above)
Comment (jetexe) [reply tests/Tests/PHPUnit/Traits/AdditionalAssertionsTraitTest.php]: Done

File: tests/Tests/PHPUnit/Traits/AdditionalAssertionsTraitTest.php (hunk 4 above)
Comment (jetexe) [reply tests/Tests/PHPUnit/Traits/AdditionalAssertionsTraitTest.php]: Done

**Comment by tarampampam** [comment]
Thx 4 your PR!

**Pull request merged**